### PR TITLE
Add `prepareCellForSkeleton` for `UITableView` and `UICollectionView`

### LINKED ISSUE #412: Configure Cell Feature
### Describe the feature or problem you’d like to solve

Our codebase is modularized and have only one dependency on a protocol modules.
We then inject the implementation from the main app, including one implementation to configure skeleton on cells. But because SkeletonView is creating UITableViewCell and passing it back directly there is no way for us to configure that cell.

### Proposed solution

Add an optional method to SkeletonCollectionDataSource protocol that would allow us to configure cell that is created by SkeletonView. This method is optional can can be ignored which assure backward compatibility. 

### Additional context

All of this works is for UICollectionView as well.

[PR Link](https://github.com/Juanpe/SkeletonView/pull/415)

### PR DESCRIPTION
**Pull request by nemanjachoco**
### Summary

This is a continuation of https://github.com/Juanpe/SkeletonView/pull/413

Fixes #412 

### Requirements
* [x] I've read and understood the [Contributing guidelines](https://github.com/Juanpe/SkeletonView/blob/main/CONTRIBUTING.md) and have done my best effort to follow them.
* [x] I've read and agree to the [Code of Conduct](https://github.com/Juanpe/SkeletonView/blob/main/CODE_OF_CONDUCT.md).

### CODE REVIEW

**Comment by nemanjachoco** [comment]
@Juanpe Sorry it is a different PR, but it is done now. Please let me know what do you think.

File: README.md [hunk 1]
```diff
@@ -205,6 +206,15 @@ There is only one method you need to implement to let Skeleton know the cell ide
  }
  ```
  
+ If you preffer to leave dequeu part to the library you can configure the cell using this method:
```
Comment (Juanpe) [inline README.md:209]: just a typo
```suggestion
If you prefer to leave the deque part to the library you can configure the cell using this method:
```

**Review by Juanpe — APPROVED** [review]
I left just a typo. The rest of the code LGTM 😄 

Good job 👌🏼

**Comment by Juanpe** [comment]
When the changes are done, I'll merge this PR and release a new version with both new features 🎉

**Comment by nemanjachoco** [comment]
Thanks! Looking forward to using this soon :)

**Pull request merged**